utils/db_utils.py

```diff
@@ -85,12 +85,14 @@ def query_confirmed_trips(start_date, end_date):
         # https://github.com/e-mission/op-admin-dashboard/issues/29#issuecomment-1530105040
         # https://github.com/e-mission/op-admin-dashboard/issues/29#issuecomment-1530439811
         # so just replacing the distance and duration with the humanized values for now
+        df['data.distance.raw'] = df['data.distance']
```

Comment:
IMHO, we should have `data.distance` as the raw value and `data.distance_miles` as humanized.

Reply:
What if we have data.distance_meters as the raw value and data.distace as humanized. The humanized can be km or miles based on config.

Reply:
What is the rationale for this suggestion?